pkg/console/subresource/deployment/deployment.go

```diff
@@ -95,6 +68,10 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 	replicas := int32(ConsoleReplicas)
 	gracePeriod := int64(30)
 	tolerationSeconds := int64(120)
+	volumeConfig := defaultVolumeConfig()
+	if hasCustomLogo(operatorConfig) {
```

Comment:
I'm going to eliminate `hasCustomLogo` and instead pass in the `canMount` bool provided by the `ValidateCustomLogo()` func as this already details out the viability of a good mount.